content/posts/newsletter-007/index.md

```diff
@@ -238,6 +238,126 @@ network play][designing_network_play].
 [designing_network_play]: https://azriel.im/will/2020/02/29/designing-network-play/
 [will_network_play]: will_network_play.png
 
+### [Oxidator][oxidator]
+
+[![Gameplay demo: two giant tank armies clashing](oxidator-play.gif)][oxidator-video-play]
+
+^ _gameplay demo (35000 units)_
+
+[Oxidator][oxidator] by [@Ruddle] is a real-time strategy game/engine
+written with Rust and WebGPU.
+It's inspired by Total Annihilation, Supreme Commander, Spring Engine,
+and Zero-k.
+
+The project's goal is to provide a modern, carefully crafted,
+minimal and highly constrained set of tools for players/designers
+to create mods without programming knowledge.
+
+Some of the current features:
+
+- Simulation:
+  working draft of flock behavior and collision detection,
+  basic health and damage computation,
+  construction and repair;
+- Rendering: basic display of a heightmap & 3D models (with instancing),
+  fxaa, screen space reflections;
+- UI: select units (picking and rectangle selection),
+  move & build orders;
+- Multiplayer: working PoC localhost tcp client/server;
+- Map editor: raise, lower, flatten, blur, noise pencil;
+- Unit editor: basic editor with joint & mesh selection and parameter editing
+  (speed, turn rate, health, etc);
+
+[![Unit editor demo: move agent's parts](oxidator-unit-editor.gif)][oxidator-video-unit-editor]
+
+^ _Demo of the unit editor_
+
+[![Map editor demo: use pencil tool to instantly create a lake and mountains](oxidator-map-editor.gif)][oxidator-video-map-editor]
+
+^ _Demo of the map editor_
+
+[oxidator]: https://github.com/Ruddle/oxidator
+[@Ruddle]: https://github.com/Ruddle
+[oxidator-video-play]: https://streamable.com/499j0
+[oxidator-video-unit-editor]: https://streamable.com/ywr44
+[oxidator-video-map-editor]: https://streamable.com/e5zdj
```

Comment:
Link is dead

Reply:
Could link to the gif instead: https://github.com/Ruddle/oxidator/raw/master/etc/map_editor.gif